equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -109,9 +91,33 @@ public <T extends CrmFormModelInterface> T convertCrmFormJsonToModel(JSONObject
         return (T) crmFormData;
     }
 
-    // TODO (): Delete this, once we have all CRM Form Files available for data-mapping
-    public JSONObject getCrmFormJson(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) {
-        return getCrmFileJson(crmFormDetailsCriteriaDTO);
+    public JSONObject convertCrmFormLinkedAttachments(JSONObject crmFileJsonObject) {
+        JSONObject linkedAttachments;
+
+        if (!crmFileJsonObject.has("linkedAttachments")) {
+            linkedAttachments = new JSONObject();
+            linkedAttachments.put("linkedAttachment", new JSONArray());
+            return linkedAttachments;
+        }
+
+        linkedAttachments = (JSONObject) crmFileJsonObject.get("linkedAttachments");
+
+        if (!linkedAttachments.has("linkedAttachment")) {
```

Comment:
`linkedAttachment` - same as above, reuse the string variable

Reply:
done